spring-cloud-previews/google-cloud-asset-spring-starter/src/main/java/com/google/cloud/asset/v1/spring/AssetServiceSpringAutoConfiguration.java

```diff
@@ -268,6 +268,29 @@ public AssetServiceSettings assetServiceSettings(
           .batchGetEffectiveIamPoliciesSettings()
           .setRetrySettings(batchGetEffectiveIamPoliciesRetrySettings);
 
+      RetrySettings analyzeOrgPoliciesRetrySettings =
```

Comment:
Corresponding to `asset_service.proto` [change](https://github.com/googleapis/googleapis/commit/49f575833aa4b5daa1f588fd261abe8dba344267)